### DISCUSSION TITLE
Accidentally linked wrong github account to new vercel account now cannot login in to original account

### DISCUSSION THREAD

**Discussion by CiaranPearse** [vercel/community#5328] [Account Issue] [answered]
### Question

I created a new account for a client with an email address and accidentally linked my personal github account that I use to log into my own vercel account.

Now when I login with my github account it goes to the new vercel account that I created for my client and I am unable to login to my original vercel account.

I can see my other team and request access, but in order to grant access I need to log in to that teams account, which I cant because it is now only connected with the new account.

**Comment by qudo-code** [comment, marked as answer]
Try to use "sign in with email" even if that's not how you normally login with that account.
